bin/pfcmd.pl

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -T
+#!/usr/bin/perl
```

Comment:
The reason we use the taint check for pfcmd is because it is running as root.
This is done to ensure we clean up our data before using it.